app/controllers/application_controller.rb

```diff
@@ -1,5 +1,12 @@
 class ApplicationController < ActionController::Base
-  # Prevent CSRF attacks by raising an exception.
-  # For APIs, you may want to use :null_session instead.
+  before_action :configure_permitted_parameters, if: :devise_controller?
+  
   protect_from_forgery with: :exception
+
+  protected
+
+  def configure_permitted_parameters
+  	temp = [:first_name, :last_name, :email, :password, :password_confirmation]
```

Comment:
yes, in general that's right. however devise_parameter_sanitizer.for(:sign_up) << p so it goes well in block and as bonus you don't have to create additional temporal variable. Anyway, choose what you want. It's all yours :)